woocommerce/woocommerce
# Replace Facebook icon

### All Submissions:


### Changes proposed in this Pull Request:

This is a follow-up to https://github.com/woocommerce/woocommerce/pull/34259#issuecomment-1228939439.

I noticed that we have two facebook icon files for marketing:

- https://github.com/woocommerce/woocommerce/blob/945a55adbc7691fd263ee4c07bc8d3dbf932d2e0/plugins/woocommerce/assets/images/marketing/facebook.svg (the old one)
- https://github.com/woocommerce/woocommerce/blob/945a55adbc7691fd263ee4c07bc8d3dbf932d2e0/plugins/woocommerce/assets/images/marketing/facebook-icon.svg (the new one added in PR https://github.com/woocommerce/woocommerce/pull/34259)

In this PR, we replace the old one with the new one. 

The changes made in `facebook.js` in this PR is similar to what we have done with the `google.js` in PR https://github.com/woocommerce/woocommerce/pull/34471.

Screenshots below:

![image](https://user-images.githubusercontent.com/417342/188681149-bb03fee3-f342-4e70-8913-54dfe9b5b5de.png)

![image](https://user-images.githubusercontent.com/417342/188681168-5de62741-208b-4238-9a6e-0655b15ccf32.png)

### How to test the changes in this Pull Request:

1. Go to the Marketing page: `/wp-admin/admin.php?page=wc-admin&path=%2Fmarketing`.
2. In "Recommended extensions" card, you should see Facebook icon as shown in the screenshot above.
3. Install Facebook for WooCommerce plugin.
4. Go back to the Marketing page.
5. In "Installed marketing extensions" card, you should see Facebook icon as shown in the screenshot above.

### Other information:


### FOR PR REVIEWER ONLY:
